kubeluigi/__init__.py

```diff
@@ -9,40 +8,33 @@
     pod_spec_from_dict,
     run_and_track_job,
     kubernetes_client,
-    attach_volume_to_spec
+    attach_volume_to_spec,
+    FailedJob,
 )
 
+from kubeluigi.volumes import AttachableVolume
+
 from kubernetes.client import ApiClient
+from kubernetes.client.models.v1_job import V1Job
+from kubernetes.client.models.v1_pod_spec import V1PodSpec
 
-logger = logging.getLogger("luigi-interface")
+logger = logging.getLogger(__name__)
 
 
 class KubernetesJobTask:
 
+    volumes: List[AttachableVolume] = []
+
     def _init_task_metadata(self):
-        self.job_uuid = str(uuid.uuid4().hex)
-        now = datetime.utcnow()
-        self.uu_name = "%s-%s-%s" % (
-            self.name,
-            now.strftime("%Y%m%d%H%M%S"),
-            self.job_uuid[:16],
-        )
-    
+        self.uu_name = self.name
```

Comment:
does this have any implications in lets say recurrent tasks with different parameters running simultaniously? i.e:  two `TrainClassiffier` tasks running with different parameters.